Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -431,6 +437,23 @@ define([
             get : function() {
                 return this._hasAlphaChannel;
             }
+        },
+
+        /**
+         * A flag indicating whether this imagery provider should be able to pick features - if it's set to false,
+         * UrlTemplateImageryProvider#pickFeatures will always return undefined.
+         *
+         * @memberof UrlTemplateImageryProvider.prototype
+         * @type {Boolean}
+         * @default true
+         */
+        enablePickFeatures : {
```

Comment:
Because this property has both a setter and a getter, and they just pass through to an underlying field, you don't need it at all.  Just define a public field in the constructor.